src/LibraryViewExtensionWebView2/web/library/library.html

```diff
@@ -194,6 +195,9 @@
 
         refreshLibraryView(libController);
 
+        //initial state
+        libController.setHostContext("home")
```

Comment:
@Enzo707  think this line needs to be moved to librarie.js->library.html file once you consume the npm package inside Dynamo.